spec/requests/posts/new_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+describe 'GET /posts/new' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.